molecule/builder/Dockerfile

```diff
@@ -1,13 +1,33 @@
-ARG IMAGE
-FROM $IMAGE
+# April 28th, 2018
```

Comment:
Nit: Can you include the version you pulled to get this sha? E.g., `trusty-2018-04-28`. I find this makes it easier to track things down when I have to revisit dockerfiles.